Doc/library/stdtypes.rst

```diff
@@ -4262,28 +4267,40 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
    value)`` pairs. Order comparisons ('<', '<=', '>=', '>') raise
    :exc:`TypeError`.
 
-   Dictionaries preserve insertion order.  Note that updating a key does not
-   affect the order.  Keys added after deletion are inserted at the end. ::
+   Dict preserves insertion order and are reversible.  Note that updating key
```

Comment:
I think I would keep this text as it was before, and add a new short block to say “Dictionaries are reversible” and show the code examples.